app/Data/migrations/versions/8cabb52c596a_picture_like_table.py

```diff
@@ -8,17 +8,47 @@
 from alembic import op
 import sqlalchemy as sa
 
+# revision identifiers, used by Alembic.
+from sqlalchemy.engine.reflection import Inspector
+
+revision = '8cabb52c596a'
+down_revision = '387d4c07cc85'
+branch_labels = None
+depends_on = None
+
+
+"""picture_like_table
+
+Revision ID: 8cabb52c596a
+Revises: 387d4c07cc85
+Create Date: 2021-01-04 14:22:27.553965
+
+"""
+from alembic import op
+import sqlalchemy as sa
 
 # revision identifiers, used by Alembic.
+from sqlalchemy.engine.reflection import Inspector
+
 revision = '8cabb52c596a'
 down_revision = '387d4c07cc85'
 branch_labels = None
 depends_on = None
 
 
 def upgrade():
-    pass
+    op.create_table(
+        'picture_like',
+        sa.Column('PictureId', sa.Integer, sa.ForeignKey('review.Id'), primary_key=True, nullable=False),
```

Comment:
``sa.Column('PictureId', sa.Integer, sa.ForeignKey('review.Id'), primary_key=True, nullable=False)``
should point to ``sa.ForeignKey('picture.Id')`` instead of  ``sa.ForeignKey('review.Id')``